docs/source/using/troubleshooting.adoc

```diff
@@ -11,6 +11,26 @@ toc::[]
 
 This section contains solutions to common problems that you might encounter while using Minishift.
 
+[[minshift-update-failed-due-to-permission-denied]]
+== Permission cause update to fail
+
+Depending on your operating system if you have minishift binary installed in location where current user does not have write permission then `minishift update` will fail.
+
```

Comment:
`minishift update` command replaces the old binary with the new binary using the current user.  So if the user does not have access permission on the file system to replace the minishift binary then the `update` will fail.